### ISSUE TITLE
Include Travis-CI integration and tests

### ISSUE THREAD

**Issue by bbockelm** [fermitools/cigetcert#16]
It occurred to me that we don't have any integration tests for `cigetcert`.

We've been able to make very good use of Travis-CI for this in other OSG projects.

To actually test the client/server interactions:
* OSG provides a package for setting up a test CA infrastructure.
* It's straightforward to start up a MyProxy server.
* I have no idea what's involved in setting up a test ECP endpoint.  I assume the Shibboleth folks must have something we can reuse?

**Comment by DrDaveD** [comment]
Yeah I have thought about doing it, and it would be nice, but I haven't been able to figure out how to to get all the pieces where I'd be able to have a scriptable test.  I'd also need kerberos credentials from somewhere.  A test ECP endpoint also implies a test Service Provider.  Fermilab & CILogon already have test instances of each, and I could probably create a test non-expiring Fermilab kerberos credential with kcron and sneak it off of the system I create it on, but I'm not sure it's a good idea to expose that to public access.

It seems too daunting to install everything in a sandbox: myproxy, ECP IdP, SP, and a fake kerberos installation.
